modules/serverless-hotfix-patch.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * install/serverless-upgrades.adoc
+
+:_content-type: CONCEPT
+[id="serverless-hotfix-patch_{context}"]
+= Serverless operator maintenance release upgrades
+
+== Latest and maintenance releases
+
+Beginning with {ServerlessProductName} 1.29, the different product versions are available as follows:
+
+* The latest release is available through the `stable` channel.
+* The maintenance release is available through its version-based channel, for example, `stable-1.29`. The maintenance release is the release prior to the latest release, for example if `stable` is 1.30, then maintenance release is `stable-1.29`.
+
+Using a version-based channel allows you to stay within a specific `x.y` release. Additionally, it prevents an upgrade to the latest version of the product, which might contain breaking changes.
+
+To switch to the maintenance release, update the channel parameter in the subscription object YAML file from `stable` to the corresponding version-based channel, such as `stable-1.29`.
+
+== Patches and hotfixes for maintenance releases
+
+Same as the stable release, the maintenance release is subject to patches and hotfixes, which help to maintain your deployment up-to-date with critical bug and security fixes.
+
+* Patches are updates that are distributed as z-releases, for example, {ServerlessProductName} 1.29.1 is the patch that provides updates made since version 1.29.0.
+* Hotfixes are fixes that require zero downtime and are used directly in production. They are different from the usual updates in that they upgrade the customer's deployed version, and not the latest released version.
++
+Hotfixes might not be available to all customers immediately. However, changes introduced by hotfixes often become available to all customers as part of a future release.
++
+When a hotfix pertaining to your deployment is available, you will be given the hotfix CatalogSource to update your subscription and obtain the hotfix.
++
+After a new operator release is available, hotfix operators deployed in clusters can also be upgraded. To use the latest GA version, modify the subscription to use the public CatalogSource instead of the hotfix one.
```

Comment:
Maybe simplify: "...available,already deployed operators with hotfixes can be upgraded"